includes/wf_crm_admin_form.inc

```diff
@@ -177,6 +177,7 @@ class wf_crm_admin_form {
     $this->form['delete']['#value'] = t('Remove Fields and Save Settings');
     $this->form['disable']['#value'] = t('Leave Fields and Save Settings');
     $this->form['cancel']['#value'] = t('Cancel (go back)');
+    $this->form['form_state'] = $this->form_state;
```

Comment:
Stashing the full FormState object on the form doesn't feel right. If we have to stash something on the form, maybe something like:

```
$this->form['#form_state_values'] = $this->form_state->getValues();
```

Or, ideally, if there were any other way to pass the necessary values to the right place without putting them on the form, that'd be best.